dist/sqlite-async.js

```diff
@@ -0,0 +1,318 @@
+"use strict";
```

Comment:
You can compare this directly with the current `.js` file. I'm not sure why Github isn't doing this, as that's what I saw locally.  Same with the test files.